templates/security/logout_other.html.twig

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Logout of Other Devices{% endblock %}
```

Comment:
This feels useful but too niche for a starter kit

Reply:
You mean this entire feature?